pulp-glue/pyproject.toml

```diff
@@ -33,6 +33,15 @@ documentation = "https://docs.pulpproject.org/pulp_cli/"
 repository = "https://github.com/pulp/pulp-cli"
 changelog = "https://docs.pulpproject.org/pulp_cli/CHANGES/"
 
+[project.entry-points."pulp_glue.plugins"]
```

Comment:
Have you opened up the corresponding PRs in our other cli-plugin repositories? They won't be registered as loaded until they have made this change.

Reply:
That is right, but i think it should be safe to fade this in slowly.